plugins/source/aws/resources/services/kms/grants_fetch.go

```diff
@@ -0,0 +1,27 @@
+package kms
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/kms/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchKmsGrants(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	k := parent.Item.(types.KeyMetadata)
+	config := kms.ListGrantsInput{KeyId: k.KeyId}
```

Comment:
List Grants takes either Key ID or Key ARN - this should be ok as if we're listing grants within each AWS Account, the key ID will work.  The key ARN is only required if we're trying to find grants in a different AWS account.  

I don't think that's something we're going to do while listing grants for keys.